1_Modules.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+import random
+
+
+def vinaigrette(start_date_str, end_date_str):
```

Comment:
add type hints, dont specify the type in the variable name - use type hints
https://docs.python.org/3/library/typing.html

Reply:
Add return types